.gitignore

```diff
@@ -26,4 +26,6 @@ replay_pid*
 .idea/
 .vscode/
 
-*.env
\ No newline at end of file
+*.env
+
+target/
```

Comment:
what's get created in target folder?

Reply:
spotless-index containing information related to formatting like time of formatting